inno_setup/gnucash-mingw64.iss

```diff
@@ -503,9 +503,15 @@ begin
     // internal exception, you'll fall down to the except block where
     // you expand the {%allusersprofile}
     Folder := ExpandConstant('{userdocs}');
-    // the {userdocs} folder expanding succeded, so let's test if the
-    // folder exists and if not, expand {%allusersprofile}
-    if not DirExists(Folder) then
+    // the {userdocs} folder expanding succeeded, so let's test if the
+    // folder exists and if not, try to expand %userprofile%\Documents.
+    // If this fails, default to the expansion of {%allusersprofile}.
+    if not DirExists(Folder) then begin
+      Folder := ExpandConstant('{%userprofile}') + '\Documents';
```

Comment:
It seems unlikely to me that `%USERPROFILE%\Documents` would be locale-independent, you should wrap it for translation with a [custom message](http://www.jrsoftware.org/ishelp/topic_custommessagessection.htm).

Reply:
Let's go back to the original version of proc `GetDocPath`, since `ExpandConstant('{userdocs}')`   should cover the case `%USERPROFILE%\Documents` case, too.  At least the file explorer on my German Win10 box does understand   `%USERPROFILE%\Documents`. ... but never mind, we might not need it anyways.